src/WebJobs.Script/OutOfProc/Http/HttpWorkerContext.cs

```diff
@@ -8,9 +8,11 @@ namespace Microsoft.Azure.WebJobs.Script.OutOfProc.Http
     // Arguments to start a worker process
     internal class HttpWorkerContext : WorkerContext
     {
+        public int Port { get; set; }
+
         public override string GetFormatedArguments()
         {
-            return $" --workerId {WorkerId} --requestId {RequestId}";
+            return $" --workerId {WorkerId} --requestId {RequestId} --port {Port}";
```

Comment:
TODO: Set port as an environment variable on the child process. Key: `FUNCTIONS_WORKER_PORT`

Reply:
@paulbatum  - this is now addressed. Env var `FUNCTIONS_WORKER_PORT` is set on the http worker child process